src/theme/Navbar/index.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import NavbarLayout from "@theme/Navbar/Layout";
+import NavbarComponent from "../../../docusaurus.config";
+import sidebars from "../../../sidebars";
+import axios from "axios";
+
+export default function Navbar() {
+  const [dynamicClass, setDynamicClass] = React.useState(false);
+  const [email, setEmail] = React.useState("");
+  const [sideBars, setSideBars] = React.useState();
+  console.log(sidebars, "Hey");
+  React.useEffect(() => {
+    setSideBars(sidebars.docs);
+  }, []);
+  const headers = {
+    "Content-Type": "text/plain",
+  };
+  return (
+    <NavbarLayout>
+      <div className="container">
+        <div className="header_main">
+          <div className="header_logo">
+            <img src="https://uploads-ssl.webflow.com/6098c4e3255a0a67635c1de5/6098c4e3255a0ac4e35c1e05_poly-key-logo.svg" />
+          </div>
+          <div className="header_menu">
+            <ul>
+              {NavbarComponent.themeConfig.navbar.items.map((data, index) => {
+                return data.hasOwnProperty("src") ? (
+                  <li className="github_main">
+                    <a href="https://github.com/MatrixAI/PolyKey/discussions">
+                      <img src={data.src} alt={data.alt} />
+                    </a>
+                  </li>
+                ) : (
+                  <li id="index">
+                    <a href={data.href}>{data.label}</a>
+                  </li>
+                );
+              })}
+              <li className="email_box">
+                <input
+                  type="email"
+                  placeholder="Your Email"
+                  onChange={(e) => {
+                    setEmail(e.target.value);
+                  }}
+                />
+                <button
+                  onClick={() => {
+                    axios
+                      .post(
+                        "https://matrix.us9.list-manage.com/subscribe/post?u=c95e0a682d8937f12732b48d4&amp;id=863aebab0f",
+                        {
+                          EMAIL: email,
+                        },
+                        { headers: headers }
+                      )
+                      .then((res) => {
+                        console.log(res);
+                      })
+                      .catch((err) => {
+                        console.log(err);
+                      });
+                  }}
+                >
+                  Get Started
+                </button>
```

Comment:
This should be using `action` on the form.

Reply:
https://imgur.com/DDyyMPq